domains/family22.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "sinazlf",
+        "email": "sina810@gmail.com"
+    },
+    "record": {
+        "CNAME": "d3.serveri.shop"
```

Comment:
your website is offline, cloudflare page shows
"Web server is down. Error code 521"